src/ucp/wireup/wireup_cm.c

```diff
@@ -499,32 +515,30 @@ static unsigned ucp_cm_client_uct_connect_progress(void *arg)
     ucp_wireup_cm_ep_cleanup(ep);
 
     for (i = 0; i < ucs_static_array_size(ep_init_flags_prio); ++i) {
-        ep_init_flags = ep_init_flags_prio[i];
-        ep_pack_flags = ep_get_pack_flags(worker, cm_wireup_ep, ep_init_flags);
+        ep_pack_flags = ucp_wireup_cm_address_pack_flags(worker, cm_wireup_ep,
```

Comment:
init it later - only right before it's used

Reply:
Done